django_cloud_tasks/connection.py

```diff
@@ -1,8 +1,9 @@
-import googleapiclient.discovery
-import os.path
 import hashlib
+import os.path
 import tempfile
 
+import googleapiclient.discovery
```

Comment:
this goes away, replace with new library

Reply:
fixed